typings/app.d.ts

```diff
@@ -0,0 +1,8 @@
+declare global {
+    export type Nullable<T> = T | null;
```

Comment:
сюда еще можно дописать undefined. Undefined так же как и null означает отсутствие значение, разница в том, что undefined задается неявно, а null - явно (например если в функции не указать return, она неявно вернет undefined)
```suggestion
    export type Nullable<T> = T | null | undefined;
```